include/dt-bindings/usb/usb.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2019 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_USB_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_USB_H_
+
+#define USB_MAXIMUM_SPEED_LOW_SPEED	0
```

Comment:
I would move those to `usb_dc.h` or similar and then have some code that translates the string into a variable containing one of the values. I would do this directly in PR #14977

Reply:
we need this kind of stuff 
b441bba68a64a3ba8a6f389b4845b8878d632921

Reply:
And especially here 6098067af34951b0ef7bf2ab1dae2721213f49a1. This is just the example, should be changed to every place we define endpoints.